spec/FCM.spec.js

```diff
@@ -348,22 +343,19 @@ describe('FCM', () => {
         keyAgain: 'valueAgain',
       };
 
-      const pushId = 'pushId';
-      const timeStamp = 1454538822113;
-      const timeStampISOStr = new Date(timeStamp).toISOString();
+      const unusedPushId = 'pushId';
+      const unusedTimeStamp = 1454538822113;
```

Comment:
Why the name change to "unused"?

Reply:
The `push_id` and `time` parameters of payload, are used only on Android so in iOS tests I renamed them to indicate that they are not relevant for this test.

Reply:
Could you please remove the "unused" from the var names and add a code comment instead? Future contributors would have to guess what "unused" means and may jump to wrong conclusions.

Reply:
sure, not a problem